WordPressVIPMinimum/Sniffs/SVG/HTMLCodeSniff.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * WordPressVIPMinimum Coding Standard.
+ *
+ * @package VIPCS\WordPressVIPMinimum
+ */
+
+namespace WordPressVIPMinimum\Sniffs\SVG;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Util\Tokens;
+
+/**
+ * This function generates an error when
+ * <?php or <? is found in a SVG file.
+ *
+ * The aim is to bring this to the notice of
+ * reviewers, so they can make sure the PHP-code
+ * is safe. The reason for this is that SVG files
+ * are often included in PHP files, but manually
+ * scanning SVG files takes alot of time and is prone
+ * to errors.
+ */
+class HTMLCodeSniff implements Sniff {
+
+	/**
+	 * Returns the token types that this sniff is interested in.
+	 *
+	 * We want everything function-related.
+	 *
+	 * @return array(int)
+	 */
+	public function register() {
+		return array(
+			T_OPEN_TAG,
+		);
+	}
+
+	/**
+	 * Processes the tokens that this sniff is interested in.
+	 *
+	 * @param \PHP_CodeSniffer\Files\File $phpcsFile The file where the token was found.
+	 * @param int                         $stackPtr  The position in the stack where
+	 *                                               the token was found.
+	 *
+	 * @return void
+	 */
+	public function process( File $phpcsFile, $stackPtr ) {
+		// Get tokens.
+		$tokens = $phpcsFile->getTokens();
+
+		$nxt = $phpcsFile->findNext(
+			T_OPEN_TAG,
```

Comment:
the `process` method is entered only when `T_OPEN_TAG` is encountered. That say, mere entering of this method makes it explicit that an open tag has been found, and there is no need to check whether it really is an open tag, or of what type.

Thus, in case we are in this method, we can directly return the Warning.